packages/quicktype-core/src/language/Swift.ts

```diff
@@ -914,6 +921,18 @@ encoder.dateEncodingStrategy = .formatted(formatter)`);
         });
     }
 
+    private emitPreviewContentExtension(c: ClassType, className: Name): void {
+        this.emitBlockWithAccess(["extension ", className], () => {
+            this.emitLine("static let preview = ", className, "(");
+            this.indent(() => {
+                this.forEachClassProperty(c, "none", (name, _, _p, position) => {
+                    this.emitLine(name, ": ", "value", position !== "only" && position !== "last" ? "," : "");
```

Comment:
I made a start with emitting the code. However I'm a little lost on how to get the source value here?